packages/dx-grid-core/src/plugins/table-column-resizing/reducers.js

```diff
@@ -3,11 +3,9 @@ const MIN_SIZE = 40;
 export const changeTableColumnWidths = (state, { shifts }) => {
   const { columnWidths } = state;
   const nextColumnWidth = columnWidths.slice();
-  let index;
-  const updatedColumn = nextColumnWidth.find((elem, elemIndex) => {
-    index = elemIndex;
-    return !!shifts[elem.columnName];
-  });
+  const currentShift = elem => !!shifts[elem.columnName];
+  const updatedColumn = nextColumnWidth.find(currentShift);
+  const index = nextColumnWidth.findIndex(currentShift);
```

Comment:
There is no need to use find & findIndex. Find an index first. After that use the index to get data in the array.